Makefile

```diff
@@ -0,0 +1,54 @@
+NAME=go-md2man
+BINARYNAME=md2man
+OWNER=cpuguy83
+ARCH=$(shell uname -m)
+VERSION=0.0.0
+
+build: build-darwin build-linux
```

Comment:
Can we call this `cross` instead?
We can add a build target, but it should only be `go build`